packages/datadog-plugin-http/test/client.spec.js

```diff
@@ -221,6 +221,31 @@ describe('Plugin', () => {
               })
             })
           })
+
+          it('should support configuration as an WHATWG URL object', done => {
```

Comment:
This test case can be written as an async function, which can be much cleaner here, since we're dealing with promises all the way through it. This pretty much applies to the whole file, so if you'd prefer to match the existing style, that's fine too. [Non-blocking comment.]

Reply:
makes sense to me, i gave moving this over to an async function a shot (which I think basically means being able to return a promise vs having to pass in `done` ?). i think it's alright but let me know if the syntax is correct here.

Reply:
@bengl Since this was your only comment, I'll go ahead and merge this since it's been addressed along with the other requested changes. Please let me know if I missed something.